src/AtelierTomato.Markov.Storage/ISentenceAccess.cs

```diff
@@ -5,23 +5,25 @@ namespace AtelierTomato.Markov.Storage
 	public interface ISentenceAccess
 	{
 		/// <summary>
-		/// Read a random <see cref="Sentence"/> that matches the given <paramref name="filter"/> and includes the given <paramref name="keyword"/>.
+		/// Read a random <see cref="Sentence"/> that matches the given <paramref name="filter"/>,  are allowed to access from the <paramref name="origin"/>, and includes the given <paramref name="keyword"/>.

```

Comment:
I don't like the name origin. This isn't really an origin, it is more of a destination? QueryScope? Something like that?